Support monitor helm chart (#15803)
Alluxio/alluxio: integration/kubernetes/helm-chart/monitor/README.md

# Introduction

This chart bootstraps a monitoring system on a [Kubernetes]() cluster using the [Helm]() package manager. This monitor system can be used to 

> monitor an alluxio cluster started on [Kubernetes]() cluster.

Reviewer: don't forget the links :P

## Pre-requisites

### Kubernetes
Kubernetes 1.11+ with Beta APIs enabled

## Install the Chart

> To install the Monitor Chart into your Kubernetes cluster :

Reviewer: ```suggestion
To install the Monitor Chart into your Kubernetes cluster:
```

```
$ helm install --namespace "alluxio" "alluxio-monitor" monitor
```

After installation succeeds, you can get a status of Chart

```
$ helm status "alluxio-monitor"
```

## Uninstall the Chart

If you want to delete your Chart, use this command:

```
$ helm delete --purge "alluxio-monitor"
```

## Configuration
The monitor system is implemented based on Prometheus + Grafana, the resource files are placed in the `monitor/source` directory.

> Before installing monitor, make some appropriate modifications.

Reviewer: ```suggestion
Before installing the monitor chart, you may make some appropriate modifications to the configuration.
```

### 1. source/grafana/datasource.yaml
This grafana datasource url domain name is `[MONITORNAME]-prometheus`, for example: our monitor installation name is `alluxio-monitor`, then it will be 'alluxio-monitor-prometheus'  
```
datasources:
  - name: Prometheus
    ...
    url: http://alluxio-monitor-prometheus:9090 
```
### 2. source/prometheus/prometheus.yaml
Change each prometheus job's namespace, For example, if the alluxio cluster we want to monitor is installed in `alluxio` namespace, then edit the prometheus.yaml:
```
scrape_configs:
  - job_name: 'alluxio master'
    kubernetes_sd_configs:
      - role: pod
        namespaces:
          names:
            - alluxio
```
### 3. Enable the alluxio metrics

> To use the monitor, we need the alluxio prometheus podAnnotations defined in the '../alluxio/values.yaml' metrics part, so it is necessary to set metrics enable true before installing alluxio.

Reviewer: ```suggestion
To use the monitor, we need the alluxio prometheus podAnnotations defined in the '../alluxio/values.yaml' metrics part, so it is necessary to enable metrics before installing the alluxio chart.
```

# Introduction

This chart bootstraps a monitoring system on a [Kubernetes]() cluster using the [Helm]() package manager, this chart is used to monitor alluxio cluster.

## Prerequisites

### Kubernetes
Kubernetes 1.11+ with Beta APIs enabled

## Install the Chart

To install the Monitor Chart into your Kubernetes cluster :

```
helm install --namespace "alluxio" --name "monitor" monitor
```

After installation succeeds, you can get a status of Chart

```

> helm status "monitor"

Reviewer: ```suggestion
$ helm status "monitor"
```

```

## Uninstall the Chart

If you want to delete your Chart, use this command:

```

> helm delete  --purge "monitor"

Reviewer: ```suggestion
$ helm delete  --purge "monitor"
```
